apps/recnet-api/src/database/prisma/prisma.connection.provider.ts

```diff
@@ -0,0 +1,53 @@
+import { execSync } from "child_process";
+
+import { Injectable, OnModuleInit } from "@nestjs/common";
+import { PrismaClient } from "@prisma/client";
+
+const postgresConfig = {
+  host: process.env.RDS_HOSTNAME,
+  port: parseInt(process.env.RDS_PORT, 10),
+  database: process.env.RDS_DB_NAME,
+  username: process.env.RDS_USERNAME,
+  password: process.env.RDS_PASSWORD,
+};
```

Comment:
[nit]: you can use `zod` to parse the process.env. For example:
```
import { z } from "zod"
const PostgreConfigSchema = z.object({
    host: z.string(),
    port: z.number(),
    database: z.string(),
    username: z.string(),
    password: z.string(),
})

const postgresConfig = PostgreConfigSchema.parse(process.env)
```

The error message would be better since `zod` would tell you which field exactly is invalid or unexpected, instead of error like this `PrismaClientInitializationError: The provided database string is invalid. Error parsing connection string: invalid port number in database URL. Please refer to the documentation in https://www.prisma.io/docs/reference/database-reference/connection-urls for constructing a correct connection string. In some cases, certain characters must be escaped. Please check the string for any illegal characters.` It makes it easier to debug if any env var is missing!

Reply:
Same thing apply to `process.env.DB_MIGRATE` & `process.env.PRISMA_SCHEMA`. Personally, I always use `zod` to parse the whole env variables, use `EnvSchema` or something to prevent me directly accessing the `process.env`, it's error-prone and easy to have typos.

Reply:
I tend to use the config service provided by [nestJS](https://docs.nestjs.com/techniques/configuration) to deal with these configs.  I will create another PR to enhance the configuration management and will consider to take in this advice.